manifests/k/kmeleonbrowser/org/K-Meleon/75.1/kmeleonbrowser.org.K-Meleon.installer.yaml

```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: kmeleonbrowser.org.K-Meleon
+PackageVersion: 75.1
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft
+  InstallerUrl: https://versaweb.dl.sourceforge.net/project/kmeleon/k-meleon/75.1/K-Meleon75.1.exe

```

Comment:
```suggestion
  InstallerUrl: https://sourceforge.net/projects/kmeleon/files/k-meleon/75.1/K-Meleon75.1.exe/download
```

Reply:
That isnt a direct download link, what i put is.

Reply:
The one you put is a mirror, which will not always work for everyone.

Reply:
Alright it looks like it works anyway though.